src/container/HomeContainer.jsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import Home from "../components/Home/Home";
+import usersData from '../data/users.json';
+
+const HomeContaienr = () => {
```

Comment:
I think stylistically it would be better if we delete this component and put all the stuff here inside the Home component. instead of passing down props.